### PR TITLE
Centralize server settings

### PR DESCRIPTION
**Pull request by esimonov**
Centralize server settings and implement fetching database config.

### CODE REVIEW

File: cmd/dynamodb/database-config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+type: dynamodb
```
Comment (erudenko) [inline cmd/dynamodb/database-config.yaml]: do you think we need a different file for database config?

File: cmd/demo/server-settings.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+staticFolderPath: ./static
+emailTemplatesPath: ./email_templates
+emailTemplateNames:
+  welcome: welcome.html
+  resetPassword: reset_password.html
+  verifyEmail: verify_email.html
+
+accountConfigPath:  ./account-config.yaml
+
+appsImportPath: ./apps.json
```
Comment (erudenko) [inline cmd/demo/server-settings.yaml]: I'm not sure it's part of app config

File: cmd/embedded/database-config.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+type: boltdb
```
Comment (erudenko) [inline cmd/embedded/database-config.yaml]: I'm not sure we need a dedicated config file for a database config

File: cmd/mgo/database-config.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+type: mongodb
```
Comment (erudenko) [inline cmd/mgo/database-config.yaml]: we don't need dedicated file congif for database

File: cmd/demo/server-settings.yaml (hunk 2 above)
Comment (esimonov) [reply cmd/demo/server-settings.yaml]: Agreed.

File: server/server-config.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+host: http://localhost:8080
```
Comment (erudenko) [inline server/server-config.yaml]: we need to add documentation in config file

Comment (esimonov) [reply server/server-config.yaml]: gonna add in the forthcoming PR, this one has grew too large :)

**Pull request merged**